hc/accounts/tests/test_check_token.py

```diff
@@ -21,8 +21,50 @@ def test_it_redirects(self):
         self.profile.refresh_from_db()
         self.assertEqual(self.profile.token, "")
 
-    ### Login and test it redirects already logged in
+    def test_login_redirects_when_already_logged_in(self):
+        """ Ensure that when a logged in user visits login page,
+        he is redirected to his profile page
+        """
+        # login
+        r = self.client.post("/accounts/check_token/alice/secret-token/")
```

Comment:
use `reverse`
you can find more information about the same from the Django documentation

Reply:
Done